test/e2e_node/e2e_node_suite_test.go

```diff
@@ -54,6 +54,7 @@ var e2es *services.E2EServices
 var runServicesMode = flag.Bool("run-services-mode", false, "If true, only run services (etcd, apiserver) in current process, and not run test.")
 var runKubeletMode = flag.Bool("run-kubelet-mode", false, "If true, only start kubelet, and not run test.")
 var systemValidateMode = flag.Bool("system-validate-mode", false, "If true, only run system validation in current process, and not run test.")
+var enableCriFlag = flag.Bool("enable-cri", false, "If true, test will run in CRI mode and avoid docker-specific assumptions.")
```

Comment:
Sorry, that is a legacy code we missed when refactoring.

Previously, we were also using `enable-cri` to enable and disable cri integration in node e2e. However, later we added the `kubelet-flags` flag, and passed all kubelet flags through it including the `experimental-cri` flag.